### ISSUE TITLE
Measurement data cannot be reported due to modification of tenant token

### ISSUE THREAD

**Issue by ATM006** [sitewhere/sitewhere#847]
I have created a tenant whose token is "ABC" and can report the measurement data normally. After modifying the tenant token to "ABCD", the measurement data cannot be reported! Can you help me? Help me

**Comment by ATM006** [comment]
old Topic：SiteWhere/ABC/input/json
new Topic：SiteWhere/ABCD/input/json

**Comment by derekadams** [comment]
Yes, the default topic naming convention uses the tenant token to differentiate the data. This allows the same configuration to be portable across multiple tenants. You can update the event source configuration to a hard-coded value if you don't want it to change though.
